src/main/kotlin/me/ezra_home/retail_software_solution/model/entity/FruitEntityDto.kt

```diff
@@ -0,0 +1,22 @@
+package me.ezra_home.retail_software_solution.model.entity
+
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import java.io.Serializable
+import java.math.BigDecimal
+import java.time.OffsetDateTime
+import java.util.*
```

Comment:
we don't want star imports. Specify each import. You can adjust IDE settings to prevent it from adding * automatically